xapian-letor/tests/harness/testutils.h

```diff
@@ -90,4 +90,11 @@ void test_mset_order_equal(const Xapian::MSet &mset1,
 	expected_exception = NULL;\
     } while (0)
 
+/// Check that CODE throws exactly Xapian LetorParseError TYPE.
+#define TEST_PARSE_EXCEPTION(db_path, qrel, data_directory, testfile)\
+	TEST_EXCEPTION(Xapian::LetorParseError,\
+		       Xapian::prepare_training_file(db_path, data_directory +\
+						     testfile, qrel, 10,\
```

Comment:
The wrapping here is unconventional as the arguments of `+` aren't aligned vertically.  Better to wrap after `db_path,` so `data_directory + testfile,` doesn't need to be split (the macro body only needs a 4 column indent rather than the 8 currently used so it should fit).

Reply:
fixed in a fixup commit.